src/net/sourceforge/kolmafia/swingui/widget/AutoHighlightSpinner.java

```diff
@@ -18,7 +17,7 @@ public AutoHighlightSpinner() {
   }
 
   public void setValue(int value) {
-    this.setValue(IntegerPool.get(value));
+    this.setValue((Integer) value);
```

Comment:
Casts? Swing widget thing?

Reply:
My guess is that this one predates autoboxing. If you remove the case straight it just calls itself. I think removing the function entirely should work, though.